db/migrate/20220705164909_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [x] According to the ERD, one of the foreign keys in the likes table references the `author` and not `user` in the Users table. Hint 👇 

```suggestion
  def change
    add_reference :likes, :author, foreign_key: { to_table: :users }, null: false
  end
```

Reply:
thank you for your help :100: